canary/apps/react/vite3/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "useDefineForClassFields": true,
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "allowJs": true,
-    "skipLibCheck": true,
+    "skipLibCheck": false,
```

Comment:
Nit: think we can just delete `skipLibCheck` entirely

Reply:
I'd prefer to keep it so that at a glance we can tell it's disabled, rather than assuming knowledge of the `tsconfig` defaults.

Reply:
Well you do need to know TS internals to know what `skipLibCheck` is but was just a nit 😄